### ISSUE TITLE
Missing bullet point

### ISSUE THREAD

**Issue by abijames** [mgifford/a11y-courses#14]
Great resource! I believe there is a bullet point missing before "Google's Introduction to Web Accessibility https://webaccessibility.withgoogle.com/course". The page currently reads as the FutureLearn course is from Google which its not (its from a EU project MOOCAP).

**Comment by weboverhauls** [comment]
@abijames Thanks!
